pkg/sql/crdb_internal.go

```diff
@@ -2002,7 +2002,7 @@ func showCreateIndexWithInterleave(
 	semaCtx *tree.SemaContext,
 ) error {
 	f.WriteString("CREATE ")
-	idxStr, err := catformat.IndexForDisplay(ctx, table, &tableName, idx, semaCtx)
+	idxStr, err := catformat.IndexForDisplay(ctx, table, &tableName, idx, "" /* partition */, semaCtx)
```

Comment:
is it possible to partition an interleave?

Reply:
heh, no: https://github.com/cockroachdb/cockroach/issues/60699

but also the interleave ordering in SHOW CREATE is incorrect: https://github.com/cockroachdb/cockroach/issues/60701. I'll add another commit to this PR to fix that.